app/frontend/src/components/Scroll/_ChevronScroll.scss

```diff
@@ -0,0 +1,62 @@
+@use "../Basics/_colors" as *;
+@use "components/Basics" as *;
+
+.chevron-scroll-outer-container {
+  white-space: nowrap;
+  display: flex;
+  width: 1088px;
+  height: 47px;
+}
+
+.chevron-scroll-child-container {
+  scroll-snap-type: x mandatory;
+  overflow: auto;
+  display: flex;
+  align-items: center;
+  gap: 16px;
+}
+
+.chevron-scroll-child-container::-webkit-scrollbar {
+  display: none;
```

Comment:
Do we need to use the webkit? It seems like this feature is unsupported in Firefox at the moment. Any thoughts on the best way of handling this for unsupported browsers.

Reply:
I think webkit is the only way to handle hiding scroll bar for chrome, safari, and opera, but I can add `scrollbar-width: none;` to cover firefox


![image](https://github.com/hackforla/CivicTechJobs/assets/73561520/9132d9a9-3a7e-4930-93eb-59983c5b4601)

Reply:
That sounds good to me. I don't know what the feedback cycle will be now that we have our staging environment, but once this is merged in, it might be worth it to ask the team if having a scroll bar is preferable. I can see a case where a visible scroll bar would be a useful user signal that an element is scroll-able.